cityofaustin/atd-product
# 10562 redesign home page

Fixes cityofaustin/atd-data-tech/issues/10562

This issue revamps our website's home page! One thing to note is that this PR does not address functionality/design for mobile devices/ or super small screens, so we will need to create another issue to address those UI considerations. I ended up addressing the hero section problem we talked about in dev sync on 10/31 and think I found a good solution. Open to any feedback on design and code!

Test steps:

[Netlify link](https://deploy-preview-50--atd-product.netlify.app/)

1. Mostly make sure all the links work and that the carousel for the featured issues functions as expected.
2. You should be able to click each carousel item and be rerouted to the page for that project or product.
3. Check all the tabs on the navbar and footer.
